addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.js

```diff
@@ -87,7 +87,11 @@ export class AttendeeCalendarController extends CalendarController {
             if (record.rawRecord.recurrency) {
                 this.openRecurringDeletionWizard(record);
             } else {
-                super.deleteRecord(...arguments);
+                const action = this.orm.call("calendar.event", "unlink_event", [
+                    record.id,
+                    record.attendeeId,
+                    ]);
```

Comment:
```suggestion
                const action = this.orm.call("calendar.event", "unlink_event", [
                    record.id,
                    record.attendeeId,
                ]);
```
Nitpick: Format it like this